### ISSUE TITLE
Contact old Songbird localizers

### ISSUE THREAD

**Issue by AntoineTurmel** [nightingale-media-player/nightingale-l10n#10]
Once the langpack is ready to translate on Babelzilla, we should contact people who helped localizing Songbird to keep work on Nightingale translation. 

Some are mentions here :
https://github.com/nightingale-media-player/nightingale-hacking/blob/sb-trunk-oldxul/app/content/html/aboutCredits.xhtml

Else you can find all names here in translators.html :
https://github.com/nightingale-media-player/songbird-l10n

We can team up this work with Mozilla communities as this was done once before. Most of french localizers are Mozilla contributors for example.

**Comment by AntoineTurmel** [comment]
Ok some people express interest to help on Nightingale translation on forums and Twitter.
They would need at least a Babelzilla account, and if they want to be more involved, a GitHub account.

**Comment by AntoineTurmel** [comment]
It seems that "mitchmcalister" (product director of Songbird) is taking care of several locales now but there are still contributors working on their own locale, translating some useless stuff for closed-source components...

**Comment by AntoineTurmel** [comment]
Ok time to contact people now,
fr and de teams are up, rest we have to contact formers translators.

**Comment by AntoineTurmel** [comment]
Ok first part of collecting mails quite done :) I have to ask missing mails to Goofy...

**Comment by AntoineTurmel** [comment]
While I'm waiting for latest e-mail address, here is a draft of the e-mail I want to send:
https://nightingale.etherpad.mozilla.org/localizers-mail

@freaktechnik @rsjtdrjgfuzkfg @ilikenwf
Let me know what you think.

I'm missing mails for:
ગુજરાતી (Gujarati)
Монгол (Mongolian)
मराठी (Marathi)
සිංහල (Sinhala)

Should I contact matching people from Mozilla L10n teams ? (maybe It's up for a different e-mail message)
https://wiki.mozilla.org/L10n:Teams

**Comment by rsjtdrjgfuzkfg** [comment]
@GeekShadow: I did some changes (and saw freaktechik doing some as well), however I have some minor notes not fitting into etherpad:

`CC` - I'd put the translators into BCC, or write individual eMails and properly put them in the to-field.

`based on Mozilla` -  I personally would prefer powered by Mozilla (as in credits) or based on XULrunner.

**Comment by AntoineTurmel** [comment]
Yeah by CC I meant BCC sorry ^^ (not gonna leak +100 e-mails address), individual emails seems betters though. And yeah replaced "based on Mozilla".
For missing mails I think I'll do another mail message.
Also @rsjtdrjgfuzkfg now on GitHub I have changed to @AntoineTurmel ;)

**Comment by AntoineTurmel** [comment]
So if it's ok, I gonna send this message to all mails address I have.

**Comment by AntoineTurmel** [comment]
Done last sunday. Get 5 Mail delivery failed, I have to look for missed mails address.
